package.json

```diff
@@ -30,7 +30,7 @@
   "lint-staged": {
     "*.{js,ts,tsx,mdx,md}": [
       "prettier --config=.prettierrc.precommit.js --write",
-      "eslint --cache --fix"
+      "eslint --cache"
```

Comment:
eslint + prettier misformats 

````
<Blah>
```
something
```
</Blah>
````

This is the same fix I applied to mdx-docs: https://github.com/xataio/mdx-docs/blob/main/package.json#L32